pkg/server/api/localauthority/v1/service_test.go

```diff
@@ -0,0 +1,1024 @@
+package localauthority_test
+
+import (
+	"context"
+	"crypto"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/andres-erbsen/clock"
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	localauthorityv1 "github.com/spiffe/spire-api-sdk/proto/spire/api/server/localauthority/v1"
+	"github.com/spiffe/spire/pkg/common/telemetry"
+	"github.com/spiffe/spire/pkg/server/api"
+	localauthority "github.com/spiffe/spire/pkg/server/api/localauthority/v1"
```

Comment:
Redundant alias

Reply:
it is not, if we does not set alias it will try to use `v1`

Reply:
I've tried to run the tests without this alias, and they are all passing; the package name being imported from github.com/spiffe/spire/pkg/server/api/localauthority/v1 is already "localauthority".